pkg/volume/plugins.go

```diff
@@ -455,6 +455,7 @@ type Spec struct {
 	Volume           *v1.Volume
 	PersistentVolume *v1.PersistentVolume
 	ReadOnly         bool
+	InlineVolume     bool
```

Comment:
Where is `InlineVolume` set?

Reply:
same question, I also forgot again why we need an `InlineVolume` bool instead of just seeing that the `Volume` field is set (in my mind they should mean the same thing). I think this bit is confusing enough to add to the design doc. Could you add this to the design (maybe just take a note and do it later since codefreeze is soon).

Reply:
@davidz627 this is needed to differentiate between the case when `Spec` holds a regular PV vs the migration code paths where there is a place-holder PV for an inline-volume. I will add this to the design document.

Reply:
@jsafrane latest commit has the code to set `InlineVolume`